Dockerfile.sdk

```diff
@@ -160,12 +165,29 @@ RUN cmake -DCMAKE_INSTALL_PREFIX=/workspace/install \
           -DTRITON_ENABLE_PYTHON_GRPC=ON \
           -DTRITON_PACKAGE_PERF_ANALYZER=ON \
           -DTRITON_ENABLE_GPU=${TRITON_ENABLE_GPU} \
-          /workspace/perf_analyzer
-RUN make -j16 perf-analyzer python-clients
-
-RUN pip3 install build \
-    && cd /workspace/perf_analyzer/genai-perf \
-    && python3 -m build --wheel --outdir /workspace/install/python
+        /workspace/perf_analyzer && \
+        make -j16 perf-analyzer python-clients && \
+        pip3 install build && \
+        cd /workspace/perf_analyzer/genai-perf && \
+        python3 -m build --wheel --outdir /workspace/install/python; \
+    else \
+        ls /workspace/perf_analyzer/ && \
+        tar -xzf /workspace/perf_analyzer/perf_analyzer*.tar.gz -C /workspace/install/bin && \
+        echo "Perf Analyzer binaries was extracted and not build" && \
+        cmake -DCMAKE_INSTALL_PREFIX=/workspace/install \
+          -DTRITON_VERSION=`cat /workspace/TRITON_VERSION` \
+          -DTRITON_REPO_ORGANIZATION=${TRITON_REPO_ORGANIZATION} \
+          -DTRITON_COMMON_REPO_TAG=${TRITON_COMMON_REPO_TAG} \
+          -DTRITON_CLIENT_REPO_TAG=${TRITON_CLIENT_REPO_TAG} \
+          -DTRITON_ENABLE_PYTHON_HTTP=ON \
+          -DTRITON_ENABLE_PYTHON_GRPC=ON \
+          -DTRITON_PACKAGE_PERF_ANALYZER=ON \
+          -DTRITON_ENABLE_GPU=${TRITON_ENABLE_GPU} \
+        /workspace/perf_analyzer && \
+        make -j16 python-clients && \
+        mkdir -p /workspace/install/python && \
+        cp /workspace/perf_analyzer/genai_perf-*.whl /workspace/install/python/; \
+    fi
```

Comment:
Looking back and by having a knowledge that PA is using `client` source code, I'm kind of loosing purpose of this condition. 
Mean may be we can simply build out of PA repository by building all the targets from "client", somehow it create a feeling that in this case we are  circle back to previous process of `client` repo (before we decouple it) with this permutation.